deploy/gcp/tidbclusters.tf

```diff
@@ -1,32 +1,24 @@
-data "local_file" "kubeconfig" {
-  depends_on = [module.tidb-operator.cluster_id]
-  filename   = module.tidb-operator.kubeconfig_path
-}
-resource "local_file" "kubeconfig" {
-  depends_on = [module.tidb-operator.cluster_id]
-  content    = data.local_file.kubeconfig.content
-  filename   = module.tidb-operator.kubeconfig_path
-}
-
 provider "helm" {
   alias          = "gke"
   insecure       = true
   install_tiller = false
   kubernetes {
-    config_path = local_file.kubeconfig.filename
+    config_path = local.kubeconfig
   }
 }
+
 module "default-tidb-cluster" {
   providers = {
     helm = "helm.gke"
   }
   source                     = "../modules/gcp/tidb-cluster"
-  gcp_project                = module.tidb-operator.gcp_project
-  gke_cluster_location       = module.tidb-operator.gke_cluster_location
-  gke_cluster_name           = module.tidb-operator.gke_cluster_name
+  cluster_id                 = module.tidb-operator.cluster_id
+  gcp_project                = var.GCP_PROJECT
+  gke_cluster_location       = var.GCP_REGION
```

Comment:
`var.GCP_REGION` must be updated to `local.location` if https://github.com/pingcap/tidb-operator/pull/809 is merged.